targets/x86-geode/profiles.mk

```diff
@@ -0,0 +1,8 @@
+X86_GENERIC_NETWORK_MODULES := kmod-3c59x kmod-e100 kmod-natsemi kmod-ne2k-pci kmod-pcnet32 kmod-8139too kmod-r8169 kmod-sis900 kmod-tg3 kmod-via-rhine kmod-via-velocity kmod-forcedeth
```

Comment:
Our geode boards have mini pci slots and usb, so maybe we need additional modules. I also agree with your proposal to use a consistent naming.